packages/flutter_tools/lib/src/runner/flutter_command_runner.dart

```diff
@@ -173,10 +175,10 @@ class FlutterCommandRunner extends CommandRunner {
     if (globalResults['version']) {
       flutterUsage.sendCommand('version');
       printStatus(FlutterVersion.getVersion(Cache.flutterRoot).toString());
-      return new Future<int>.value(0);
+      return 0;
     }
 
-    return super.runCommand(globalResults);
+    return await super.runCommand(globalResults);
```

Comment:
The super function returns a `Future<Foo>`, but we're in an async block so conceptually we're returning a `Foo`. Hence the await. I don't think it practice it makes any difference, I just like my types to match. :-)